### PR TITLE
update config in export.py

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run tests

File: tools/export.py [hunk 1]
```diff
@@ -238,7 +239,7 @@ def main(args):
     with torch.no_grad():
         export_to_onnx(model, fake_data, export_name=onnx_model_path, opset=args.opset,
                        alt_ssd_export=getattr(args, 'alt_ssd_export', False),
-                       target=args.target, verbose=True)
+                       target=args.target, verbose=False)
```
Comment (druzhkov-paul) [inline tools/export.py:247]: :pray:

File: mmdet/apis/inference.py [hunk 2]
```diff
@@ -29,6 +29,8 @@ def init_detector(config, checkpoint=None, device='cuda:0'):
     elif not isinstance(config, mmcv.Config):
         raise TypeError('config must be a filename or Config object, '
                         f'but got {type(config)}')
+    if update_config:
+        config.merge_from_dict(update_config)
```
Comment (druzhkov-paul) [inline mmdet/apis/inference.py]: Is it possible to load config and update it at a script level, not inside the API code?

Comment (Ilya-Krylov) [reply mmdet/apis/inference.py]: done

**Comment by Ilya-Krylov** [comment]
run tests

**Review by druzhkov-paul — APPROVED** [review]

**Pull request merged**